app/serializers/component_serializer/heading1_component_serializer.rb

```diff
@@ -9,6 +9,9 @@ class Heading1ComponentSerializer < ComponentSerializer::BaseComponentSerializer
     # @param [String] content of the context, this could be a string or a translation key.
     # @param [String] data for the context, for example this could be an integer or url which can be passed into a translation in the front end.
     # @param [Boolean] this boolean indicated if the context is only avalibale to screen readers.
+    #
+    # @example Initialising a heading1 component
+    #   For content: ComponentSerializer::Heading1ComponentSerializer.new(subheading_content: 'string or translation key', subheading_data: {"link": 'link.com'}, heading_content: 'string or translation key', heading_data: {"link": 'link.com'}, context_content: 'string or translation key', context_data: {"link": 'link.com'}, context_hidden: true).to_h
```

Comment:
How does this render in YARD's generated HTML? All examples should be valid Ruby Code, you'd be better off changing the @example like to say `Initialising a heading 1 component with content`

Then removing `From content:` from the example